gossip/src/cluster_info.rs

```diff
@@ -2950,7 +2958,7 @@ impl Node {
             )
             .unwrap();
         let tpu_forwards_quic =
-            bind_more_with_config(tpu_forwards_quic, QUIC_ENDPOINTS, quic_config).unwrap();
+            bind_more_with_config(tpu_forwards_quic, DEFAULT_QUIC_ENDPOINTS, quic_config).unwrap();
```

Comment:
Do we want to make these configurable as well?

Reply:
Good catch. yes. We should and I will change it.